src/content/about/index.md

```diff
@@ -31,14 +32,14 @@ Data can now be received daily, instead of weekly, monthly or quarterly, generat
 
 ### What do we do?
 
-Analytics engineers start with the raw data from source systems provided by data engineers.
+Analytics engineers start with the curated data provided by data engineers.
 
-We transform, cleanse and link it to produce curated data sets that are more useful and useable for analysts and data scientists, accelerating the production of statistics, MI, analysis, and data science products.
+We [denormalise](https://en.wikipedia.org/wiki/Denormalization), cleanse, link it and apply various other transformations to produce derived data sets that are more useful and useable for analysts and data scientists, accelerating the production of statistics, MI, analysis, and data science products.
```

Comment:
I've slightly reworded and expanded on this part - suggestion below:

> We work with analytical users (usually analysts, data scientists and statisticians) to understand how they use data, so that we can design and build derived tables to meet their needs.
> 
> To build derived tables we clean, join and transform the curated tables produced by data engineers, using [dimensional modelling](https://en.wikipedia.org/wiki/Dimensional_modeling) and implementing tests and documentation to ensure that the output is reliable and understandable.
> 
> The derived tables that we produce accelerate the production of statistics, management information (MI), analysis, and data science products.